packages/runtimes/hmr/src/loaders/hmr-runtime.js

```diff
@@ -81,6 +81,9 @@ if ((!parent || !parent.isParcelRequire) && typeof WebSocket !== 'undefined') {
       document.body.appendChild(overlay);
     }
   };
+  ws.onclose = function(e) {
+    console.warn('[parcel] 🚨 Connection to the HMR server was lost');
```

Comment:
I added this because the page needs to be refresh after parcel was restarted (otherwise HMR silently stops)